style/src/test/java/org/jline/style/StyleExpressionTest.groovy

```diff
@@ -26,7 +26,7 @@ class StyleExpressionTest
     @Before
     void setUp() {
         super.setUp()
-        this.underTest = new StyleExpression(new StyleResolver(source, 'test'))
+        this.underTest = new StyleExpression()
```

Comment:
Its fine asis, if/when I add tests I may adjust this back.  Fine to leave asis for now.